hack/manifests/csi/vsphere/05-example-pod.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
```

Comment:
This should really be a deployment and not a pod

Reply:
https://kubernetes.io/docs/concepts/workloads/pods/#workload-resources-for-managing-pods

Reply:
Changed to deployment

Reply:
Since this was changed to a deployment the filename and other relevant references to `pod` should be updated.